### PR TITLE
Use multiple exports in CSS plugin

### PR DESCRIPTION
**Pull request by jakearchibald**

### CODE REVIEW

**Comment by jakearchibald** [comment]
lol no this is broken. Don't review yet…

**Comment by jakearchibald** [comment]
Fixed!

File: lib/classname-plugin.js [hunk 1]
```diff
@@ -2,35 +2,41 @@ import { promises as fsp } from 'fs';
...
 
   return {
-    name: 'classname-loader',
+    name: 'classnames-loader',
     resolveId(source) {
       if (!source.startsWith(PREFIX)) {
         return;
       }
-      // Don’t do anything here
+      // Don't do anything here
```
Comment (surma) [inline lib/classname-plugin.js]: How _dare_ you

Comment (jakearchibald) [reply lib/classname-plugin.js]: haha it annoys the spell checker I have for some reason

**Pull request merged**